app/views/provider/request_password_resets/new.html.slim

```diff
@@ -1,8 +1,7 @@
-= javascript_pack_tag 'ResetPasswordPage'
+= javascript_pack_tag 'RequestPasswordPage'
 - authentication_providers = (@authentication_providers || []).map { |ap| {authorizeURL: ap.authorize_url, humanKind: ap.human_kind} }
 - flash_messages = (flash || []).map { |f| {type: f[0], message: f[1]}}
-- is_master_account = domain_account.master?
 
-div#pf-reset-page-container data-reset-props={flashMessages: flash_messages,
+div#pf-request-page-container data-request-props={flashMessages: flash_messages,
```

Comment:
IMO it's better with the args aligned, like this:
```suggestion
div#pf-request-page-container data-request-props={
  flashMessages: flash_messages,
```

Reply:
But that's a syntax error in slim